build.gradle

```diff
@@ -123,29 +128,32 @@ subprojects {
 
         azureSdkVersion = "1.2.21"
 
-        testcontainersVersion = "1.19.4"
+        testcontainersVersion = "1.19.7"
 
         testcontainersFakeGcsServerVersion = "0.2.0"
     }
 
     dependencies {
         compileOnly "org.apache.kafka:kafka-clients:$kafkaVersion"
-        compileOnly "org.apache.kafka:kafka-storage-api:$kafkaVersion"
+        compileOnly("org.apache.kafka:kafka-storage-api:$kafkaVersion")
 
+        implementation enforcedPlatform("com.fasterxml.jackson:jackson-bom:$jacksonVersion")
```

Comment:
could we add some comments on what this "enforcePlatform" does and why we needed it here?

e.g. as it's only added for jacksone, does it mean that we would need to know in advance which dependencies may conflict, so we can flag it as we are doing here?